Makefile-ponyc

```diff
@@ -280,6 +280,7 @@ ifneq ($(MAKECMDGOALS),clean)
   ifeq ($(llvm_version),3.9.1)
   else ifeq ($(llvm_version),5.0.2)
   else ifeq ($(llvm_version),6.0.1)
+  else ifeq ($(llvm_version),7.0.0) # this is what is shipped with ubuntu bionic llvm-7 package right now
```

Comment:
This shouldn't be added. It's not supported. It's not event the default.

If we are going to support this, we need to have CI for it.